nx_parallel/utils/decorators.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import asdict
+from joblib import parallel_config
+import networkx as nx
+from functools import wraps
+import os
+
+__all__ = [
+    "_configure_if_nx_active",
+]
+
+
+def _configure_if_nx_active():
```

Comment:
We agreed on this name in the last meeting. (ref. [Notes](https://hackmd.io/@Schefflera-Arboricola/By6_8lTmR))